db/migrate/20200405201006_set_default_level_to_test.rb

```diff
@@ -0,0 +1,9 @@
+class SetDefaultLevelToTest < ActiveRecord::Migration[5.2]
+  def up
+    change_column_default :tests, :level, 0
+  end
+
+  def down
+    change_column_default :tests, :level, :null
```

Comment:
насколько я понимаю, если использовать текущий синтаксис, то здесь должно быть так
```suggestion
    change_column_default :tests, :level, nil
```